client/components/header.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+// import logo from '../../server/public/images/favicon.png';
+
+export default class Header extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = null;

```

Comment:
If you aren't using state, you can just omit this line :) And since you aren't using props here, you can also remove the constructor method completely, since it's empty